src/include/network/lan/udev.rb

```diff
@@ -71,10 +72,9 @@ def update_udevd
 
     # Removes (key,operator,value) tripplet from given udev rule.
     def RemoveKeyFromUdevRule(rule, key)
-      rule = deep_copy(rule)
-      pattern = Builtins.sformat("%1={1,2}[^[:space:]]*", key)
+      pattern = /#{key}={1,2}[^[:space:]]*/
```

Comment:
also this part `[^[:space:]]` doesn't look like ruby regexp. What it actually should catch? something like `\S` ?